### PR TITLE
Update rg_replace.sh

### PR DESCRIPTION
**Pull request by Vanderscycle**
Added a quick note on how to use rgr on fish without using binaries. Essentially point to the local rg_replace.sh file. I am sure they are cleverer implementation but its simple enough.

### CODE REVIEW

File: useful_scripts/rg_replace.sh [hunk 1]
```diff
@@ -24,6 +24,14 @@
 #   3. `gs_rgr`
 #   4. `gs_rg_replace`
 
+# SIMPLE FISH IMPLEMENTATION (not using bins)
```
Comment (ElectricRCAircraftGuy) [inline useful_scripts/rg_replace.sh:27]: What do you mean: "not using bins"?

File: useful_scripts/rg_replace.sh [hunk 2]
```diff
@@ -24,6 +24,14 @@
...
+# essentially we create a function that executes rg_replace.sh (named rgr.sh in my config) 
+
+# function rgr
+#   set -l DOOTFILE_LOC ~/Documents/dotFiles/ # replace to the current location of rg_replace.sh
+#   bash "$DOOTFILE_LOC"/rgr.sh "$argv"
+# end
```
Comment (ElectricRCAircraftGuy) [inline useful_scripts/rg_replace.sh:33]: Where should someone place this function?

File: useful_scripts/rg_replace.sh [hunk 3]
```diff
@@ -24,6 +24,14 @@
 #   3. `gs_rgr`
 #   4. `gs_rg_replace`
 
+# SIMPLE FISH IMPLEMENTATION (not using bins)
+# essentially we create a function that executes rg_replace.sh (named rgr.sh in my config) 
+
+# function rgr
+#   set -l DOOTFILE_LOC ~/Documents/dotFiles/ # replace to the current location of rg_replace.sh
```
Comment (ElectricRCAircraftGuy) [inline useful_scripts/rg_replace.sh:31]: `DOOTFILE_LOC` should be `DOTFILE_LOC`, no? I recommend changing "replace" to "set", as well.

**Review by ElectricRCAircraftGuy — COMMENTED** [review]
I left some comments and questions. Thanks!

File: useful_scripts/rg_replace.sh (hunk 3 above)
Comment (Vanderscycle) [reply useful_scripts/rg_replace.sh]: My mistake, I often refer to my dotfiles as my doots for added confusion.

**Review by ElectricRCAircraftGuy — APPROVED** [review]

File: useful_scripts/rg_replace.sh (hunk 2 above)
Comment (Vanderscycle) [reply useful_scripts/rg_replace.sh]: ``` ~/.config/fish/config.fish ``` unless they placed their fish config elsewhere

File: useful_scripts/rg_replace.sh (hunk 1 above)
Comment (Vanderscycle) [reply useful_scripts/rg_replace.sh]: binaries, I haven't played much with binaries and so I just point to the .sh file when calling the function.

**Comment by ElectricRCAircraftGuy** [comment]
I updated the instructions to this: https://github.com/ElectricRCAircraftGuy/eRCaGuy_dotfiles/blob/master/useful_scripts/rg_replace.sh#L27L39

How's that? Does that still work and make sense?

**Pull request merged**